pytition/petition/templates/petition/petition_detail.html

```diff
@@ -82,130 +96,130 @@
-                          width: 100%;
-                          margin-top: 0.25rem;
-                          font-size: 80%;
-                          color: #00ff00;
-                        }
-                    </style>
-                    <form method='POST' name='petition' class='form-group'
-                          action='/petition/{{ petition.id }}/sign'>
-                        {% csrf_token %}
-                        {% for hidden_field in form.hidden_fields %}
-                            {{ hidden_field }}
-                        {% endfor %}
-
-                        {% if form.non_field_errors %}
-                            <div class="alert alert-danger" role="alert">
-                                {% for error in form.non_field_errors %}
-                                    {{ error }}
-                                {% endfor %}
-                            </div>
+                <div id="form-sticky" class="form-sticky">
+                    {% for message in messages %}
+                        <div
+                        {% if message.level == DEFAULT_MESSAGE_LEVELS.ERROR %}
+                            class="alert alert-danger" role="alert">
+                        {% elif message.level == DEFAULT_MESSAGE_LEVELS.SUCCESS %}
+                            class="alert alert-success" role="alert">
+                        {% else %}
+                            class="alert {{ message.tags }}" role="alert">
                         {% endif %}
-
-                        {% for field in form.visible_fields %}
-                            <div class="form-group has-feedback">
-                                {{ field.label_tag }}
-
-                                {% if form.is_bound %}
-                                    {% if field.errors %}
-                                        {{ field|add_class:"is-invalid" }}
-                                        <span class="oi oi-question-mark form-control-feedback invalid-feedback"></span>
-                                        {% for error in field.errors %}
-                                            <span class="invalid-feedback">
-                                                {{ error }}
-                                            </span>
-                                        {% endfor %}
+                            {{ message }}
+                        </div>
+                    {% endfor %}
+                    <div class="fields">
+                        <style>
+                            .oi-check-circle {
+                              width: 15px;
+                              margin: -25px 10px;
+                              float: right;
+                            }
+                            .oi-ban {
+                              width: 15px;
+                              margin: -25px 10px;
+                              float: right;
+                            }
+                            .oi-question-circle {
```

Comment:
are those still needed?

Reply:
> Also, with this PR applied, I think the narvar is too close to the divs under it.
> For instance on the org_dashboard page, the "New petition" Button is touching the navbar.
> Is it possible to add a `<p></p>` ? Or a `margin-bottom`? Or maybe there is a better approach?

I'll add margin or padding, anything but `<p></p>` : we must avoid using content tags to improve theme :)

Reply:
yes I thought so, thanks!